book/website/communication/peer-review/peer-review-guidance.md

```diff
@@ -0,0 +1,108 @@
+The following principles might help you:
+1. Be honest but constructive. 
+You are not at war. 
+There is no need to burn a paper to the ground.
+2. Avoid addressing the authors personally. 
+Say: “the paper could benefit from…” instead of “the authors need”.
+3. Stay close to the facts. 
+Do not speculate about reasons why the authors have made certain choices beyond the arguments stated in the paper.
+4. Take a developmental approach. 
+Any paper will contain flaws and imperfections. 
+Your job is to improve science by identifying problems and suggesting ways to repair them. 
+Think with the authors about ways they can improve the paper in such a way that it benefits collective scholarship. 
+5. Try to see beyond bad writing style and mistakes in spelling. 
+Also be mindful of disciplinary and cultural differences between the authors and yourself.
+
+**The substance of the advice**
+1. It is a good principle to begin the review report by describing the way you reviewed the paper. 
+If you searched for literature, checked the data and verified the results, ran additional analyses, state this. 
+It will allow the editor to adjudicate the review.
+2. Then give a brief overview of the paper. 
+If the invitation asks you to provide a general recommendation, state it early in the review.
+The recommendation is the first thing the editor wants to know.
+Typically, you are invited to recommend ‘reject’, ‘revise & resubmit’ – with major or minor revisions, or ‘accept’. 
+3. Use ‘reject’ only if absolutely necessary.
+Most journal editors make use of desk-rejection if the paper seems a poor fit; if the editor sent the paper out for review, she probably thinks it has the potential to be published.
+Start from the assumption that the authors have invested a great deal of time in the paper and that they want to improve it. 
+4. List the strengths and the weaknesses of the paper. 
+To ease the message you can use the sandwich principle: start with the strengths, then discuss the weaknesses, and conclude with an encouragement.
+5. Try to give actionable advice. 
+For the weaknesses in the paper, suggest ways to repair them. 
+It is convenient for both the editor and the authors to number your suggestions.
+Distinguish major issues such as not discussing alternative explanations from minor issues such as missing references and typos. 
+There could be points that the authors are underselling. 
+In that case, you could identify them as strengths that the authors can emphasize more strongly.
+It is handy to refer to issues with direct quotes and page numbers or line numbers. 
+6. Some reviewers sign their reviews. 
+A non-anonymous review can be seen an accountability device: by exposing who they are to the authors of the paper, the reviewrs set higher standards for themselves.
+There are [discussions](??? any reference here) about implementing open peer review model in research more generally.
```

Comment:
I think you need to make clear here the difference between open peer review (which can be defined as the review being open for anyone viewing the paper to see) and signed peer review (i.e the reviewer signing their name). And that you can have a blend of either- i.e not being able to see the review but seeing the name of reviewer who did it, being able to see the review but not the name of the reviewer, or being able to see both!

Reply:
awesome, added this part!